### DISCUSSION TITLE
Changelog language translations

### DISCUSSION THREAD

**Discussion by TonyGravagno** [RealRaven2000/FiltaQuilla#288] [General]
In https://github.com/RealRaven2000/FiltaQuilla/discussions/284#discussioncomment-10154052 @RealRaven2000 wrote:

> and I translate into up to 21 languages.

Try Esperanto. Or perhaps a single link to translate.google.com.

Seriously though, I speak a few languages (to varying degrees) and have a global perspective, so I understand the sentiment and concern and I'm not flippant about this.

But these days auto translations are much better than they used to be. It should be possible to just add a link to a site that comes back to scan/translate a page from any one language to any other. Unless you get significant flak about this, perhaps you can save yourself some time with an experiment for the next year: provide English-only info with a link for instant translation (no copy/paste) to any language. Source text can be on web pages that are easy for scanning. Why English and not German? That's not bias, English is simply the, um, Lingua Franca right now and (I believe) will (arguably?) reach a broader audience for this platform than others. 

Now that we have this discussion area with thousands of people participating, perhaps we can a poll about language translation. 😉   
Better, I'm guessing the Thunderbird community itself has some metrics on this and related conclusions.

**Comment by RealRaven2000** [comment]
> Seriously though, I speak a few languages (to varying degrees) and have a global perspective, so I understand the sentiment and concern and I'm not flippant about this.
> 
> But these days auto translations are much better than they used to be. It should be possible to just add a link to a site that comes back to scan/translate a page from any one language to any other. Unless you get significant flak about this, perhaps you can save yourself some time with an experiment for the next year: provide English-only info with a link for instant translation (no copy/paste) to any language. Source text can be on web pages that are easy for scanning. Why English and not German? That's not bias, English is simply the, um, Lingua Franca right now and (I believe) will (arguably?) reach a broader audience for this platform than others

I am using chat GPT because it is able to give me back the correct JSON format without translating key words, and things like serbian with cyrillic characters, I used to use google translate but that was a nightmare because I had to step through single tabs. You just have to be very persistent telling chat GPT some stuff again and again, like "please don't remove the trailing comma" or "please don't add additional language brackets" etc. it is a very dilligent but highly forgetful worker. So my prompt for the various Add-ons (which also have different sets of locales) is quite long.

**Reply by TonyGravagno** [reply]
If you wish, in August we can begin to collaborate off-list (even in Discord 😁) to create a Custom GPT that will process your source text consistently for any desired language.

**Comment by RealRaven2000** [comment]
PS: I have been thinking about adding a language dropdown to the (English only) support pages, that could definitely be done using a google API, but I am not going to mess with the UI, now that people are used to see the language that their main software Thunderbird is in. But I am always open to improvement suggestions for the translated terms, I recently accepted a big patch that [changed the expression for `account`](https://github.com/RealRaven2000/SmartTemplates/pull/302/commits)in SmartTemplates for the chinese locales zn-tw and zn-cn
